orangecontrib/text/widgets/owcorpusviewer.py

```diff
@@ -115,7 +115,8 @@ def load_features(self):
         self.selected_features = []
         if self.corpus is not None:
             self.features = [(meta.name, vartype(meta)) for meta in self.corpus.domain.metas] + \
-                            [(attr.name, vartype(attr)) for attr in self.corpus.domain.attributes]
+                            [(attr.name, vartype(attr)) for attr in self.corpus.domain.attributes] + \
+                            [(class_var.name, vartype(class_var)) for class_var in self.corpus.domain.class_vars]
```

Comment:
I know. But Orange 2 did not have `class_vars` (I guess ... at least not until very late), so there `variables = attributes + [class_var]`.

In Orange 3 this is still OK. With a single class, `attribute + [class_var] == attribute + class_vars` anyway. Algorithms that do not handle multiple classes check whether `class_var` exists (it is `None` when there are multiple classes), so `Domain.__iter__` will work fine for them. Algorithms that do handle multiple classes, would probably want to iterate accross `attribute + class_vars`, which is again OK.